openfisca_core/tools/test_runner.py

```diff
@@ -75,25 +57,28 @@ def run_tests(tax_benefit_system, paths, options = None):
     if isinstance(paths, str):
         paths = [paths]
 
-    return pytest.main([*argv, *paths] if True else paths, plugins = [OpenFiscaPlugin(tax_benefit_system, options)])
+    if options is None:
+        options = {}
```

Comment:
This does not make sense in these lines, it should be moved before the first call to `options.get`.